### ISSUE TITLE
Grenade position is wrong in default skin

### ISSUE THREAD

**Issue by NotAFile** [yvt/openspades#663]
It's floating in front of your arm. One of these days I will finally fix this.

**Comment by NotAFile** [comment]
Correction: this was because I had cg_ViewWeapon* values set. So instead, I think the ViewWeapon* translation should be applied to the arms as well

**Comment by Mduro** [comment]
My mods (unexpectedly) already account for modified cg_ViewWeapon* values because I position my arms relative to the weapon matrix anyway. Should this change be made on the skins or the game itself?

**Comment by Mduro** [comment]
Is this bug still in the latest official release (0.1.1c)? I'm not experiencing it.

**Comment by NotAFile** [comment]
yes, it was in the development release at the time of opening the issue, which is newer than 0.1.1c

**Comment by Mduro** [comment]
I think I'm misunderstanding. When I set `cg_ViewWeaponX` to `0.5`, it looks like this:

https://i.imgur.com/xLpY0sk.jpg

And the hands are attached to the grenade.
